docs/domain/analytics/index.md

```diff
@@ -0,0 +1,56 @@
+(analytics)=
+# Raw-Data Analytics
+
+**CrateDB provides real-time analytics on raw data.**
+
+In domains of real-time analytics, where you absolutely must have access to all
+the records, and can't live with any down-sampled variant, because records are
+unique, and need to be accounted for within your analytics queries, you need a
+storage system which manages all the data in its hot zone, to be available on
+your fingertips.
+
+With CrateDB, you can do all of that using plain SQL, with excellent integration
+capabilities into commodity systems using standard database access interfaces
+like ODBC or JDBC, and a proprietary HTTP interface on top.
+
+
+(bitmovin)=
+## Bitmovin Insights
```

Comment:
Presenting those details concisely is very important to us. If that resonates with you, please suggest wording improvements as we go. Thank you!